GEMB/managelayers.m

```diff
@@ -35,19 +35,23 @@
 n=length(T);
 zY2=zY;
 dzMin2=zeros(size(dz));
+dzMax2=zeros(size(dz));
 
 X=1;
 Zcum = cumsum(dz); 
+
 % check if depth is too small
-dzMin2(1)=dzMin;
-for i=2:n
-    if (Zcum(i)<=zTop+Dtol)
-        dzMin2(i)=dzMin;
-        X=i;
-    else
-        dzMin2(i)=zY2*dzMin2(i-1);
-    end
-end
+ztop=(Zcum<=zTop+Dtol);

```

Comment:
Oh I see this `ztop` variable is the same thing as the `is_top_layer` variable I suggested. Given that we already have a `zTop` variable, I recommend calling it `is_top_layer`.